index.html

```diff
@@ -120,6 +120,37 @@ <h2>A project for first-time contributions</h2>
         <!-- Keep one line of space above and below your card -->
 
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
+
+         <!-- ________ *Adelakun Godspower card START ________  -->
+         <div class="card">
+          <p class="name">Adelakun Godspower David</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com" target="_blank">twitter.com/ly_nxhyper</a>
+            <i class="fab fa-github"></i>
+            <a href="https://www.github.com" target="_blank">https://github.com/Plutobby</a>
```

Comment:
Hi, you should put your account link inside the href field, and your username between the tag like below.
So that the links shows your name and directs to your account's website.

You can update the PR by committing and pushing again.

```suggestion
            <i class="fab fa-twitter"></i>
            <a href="https://www.twitter.com/ly_nxhyper" target="_blank">ly_nxhyper (or your username)</a>
            <i class="fab fa-github"></i>
            <a href="https://www.github.com/Plutobby" target="_blank">Plutobby(or your username)</a>
```